studio/components/interfaces/Authentication/Policies/PolicyEditor/PolicyDefinition.tsx

```diff
@@ -33,9 +35,38 @@ const PolicyDefinition: FC<Props> = ({
       {showUsing(operation) && (
         <div className="flex space-x-12">
           <div className="flex w-1/3 flex-col space-y-2">
-            <label className="text-scale-1100 text-base" htmlFor="policy-name">
-              USING expression
-            </label>
+            <div className="flex items-center space-x-2">
+              <label className="text-scale-1100 text-base" htmlFor="policy-name">
+                USING expression
+              </label>
+              <Tooltip.Root delayDuration={0}>
+                <Tooltip.Trigger>
+                  <IconHelpCircle className="text-scale-1100" size={16} strokeWidth={1.5} />
+                </Tooltip.Trigger>
+                <Tooltip.Content side="bottom">
+                  <Tooltip.Arrow className="radix-tooltip-arrow" />
+                  <div
+                    className={[
+                      'bg-scale-100 rounded py-1 px-2 leading-none shadow',
+                      'border-scale-200 border w-[300px] space-y-2',
+                    ].join(' ')}
+                  >
+                    <p className="text-scale-1200 text-xs">
+                      This expression will be added to queries that refer to the table if row-level
+                      security is enabled.
+                    </p>
+                    <p className="text-scale-1200 text-xs">
+                      Rows for which the expression returns true will be visible. Any rows for which
+                      the expression returns false or null will not be visible to the user (in a
+                      SELECT), and will not be available for modification (in an UPDATE or DELETE).
+                    </p>
+                    <p className="text-scale-1200 text-xs">
+                      Such rows are silently suppressed - no error is reported.
```

Comment:
Should this be "such _queries_ are silently suppressed"?

Reply:
should be rows i think - this is verbatim from postgres's documentation